templates/planet.constructions.html

```diff
@@ -40,39 +40,36 @@ <h2 class="title is-5 is-marginless">Metal mine</h2>
 					{{ else }}
 					<button formaction="/planet/{{ .Data.Planet.ID }}/metal-mine/upgrade"
 						class="is-paddingless Planet-card__button" {{ if .Data.Timer }}disabled{{ end }}>
```

Comment:
Not necessary to do in this PR but we should disable the upgrade button if there aren't enough resources. I added this to the roadmap.